FilmRatings/templates/film_list/film_list.html

```diff
@@ -7,22 +7,41 @@
     <h3>Film list with ratings</h3>
     {% if load_results %}
         <h3 style="color:{{ festival_color }}; margin-bottom: 2px">Load results</h3>
-        <small>
         {% for load_result in load_results %}
-            <p style="padding-left: 40px; padding-top: 1px; padding-bottom: 0px; margin-top: 0px; margin-bottom: 0px;">
-                {{ load_result }}
-            </p>
+            <p class="log">{{ load_result }}</p>
+        {% endfor %}
+    {% endif %}
+    {% if unexpected_errors %}
+        <h3 class="error">Unexpected error</h3>
+        {% for error in unexpected_errors %}
+            <p class="log">{{ error }}</p>
         {% endfor %}
-        </small>
     {% endif %}
     <br>
+    {% if action.new_rating %}
+        <p>
+        {% if action.old_rating == '-' %}
+            {{ current_fan }} gave <em style="color:{{ festival_color }}">{{ action.rated_film }}</em> a rating of <em style="color:{{ festival_color }}"> {{ action.new_rating }} ({{ action.new_rating_name }})</em>
+        {% elif action.new_rating == '0' %}
+            {{ current_fan }} removed rating {{ action.old_rating }} of <em style="color:{{ festival_color }}">{{ action.rated_film }}</em>
+        {% else %}
+            {{ current_fan }} changed rating {{ action.old_rating }} of <em style="color:{{ festival_color }}">{{ action.rated_film }}</em> into <em style="color:{{ festival_color }}"> {{ action.new_rating }} ({{ action.new_rating_name }})</em>
+        {% endif %}
```

Comment:
Please consider to make a (dynamic) style class for `style="color:{{ festival_color }}"`.

Reply:
Got rid of the elaborate styling by adding an `em` selector in the (dynamic) base template.